corehq/apps/linked_domain/tests/test_view_helpers.py

```diff
@@ -434,6 +435,46 @@ def test_build_keyword_view_model_with_grouped_returns_unlinkable(self):
         actual_view_model = build_keyword_view_model(keyword)
         self.assertEqual(expected_view_model, actual_view_model)
 
+    def test_build_ucr_expression_view_model_returns_match(self):
+        ucr_expression = _create_ucr_expression(self.domain, should_save=False)
+        ucr_expression.id = '100'
+
+        expected_view_model = {
+            'type': 'ucr_expression',
+            'name': f'Data Expressions and Filters ({ucr_expression.name})',
+            'detail': {'ucr_expression_id': '100'},
+            'last_update': None,
+            'can_update': True,
+            'is_linkable': True,
+        }
+
+        actual_view_model = build_ucr_expression_view_model(ucr_expression)
+        self.assertEqual(expected_view_model, actual_view_model)
+
+    def test_build_ucr_expression_view_model_with_none_returns_none(self):
+        ucr_expression = None
+        actual_view_model = build_ucr_expression_view_model(ucr_expression)
+        self.assertIsNone(actual_view_model)
+
+    # def test_build_keyword_view_model_with_empty_returns_none(self):
+    #     keyword = {}
+    #     actual_view_model = build_keyword_view_model(keyword)
+    #     self.assertIsNone(actual_view_model)
+
+    # def test_build_keyword_view_model_with_grouped_returns_unlinkable(self):
+    #     keyword = _create_keyword(self.domain, name='keyword-test', is_grouped=True)
+    #     expected_view_model = {
+    #         'type': 'keyword',
+    #         'name': 'Keyword (keyword-test)',
+    #         'detail': {'keyword_id': f'{keyword.id}'},
+    #         'last_update': None,
+    #         'can_update': True,
+    #         'is_linkable': False,
+    #     }
+
+    #     actual_view_model = build_keyword_view_model(keyword)
+    #     self.assertEqual(expected_view_model, actual_view_model)
```

Comment:
Assume this accidentally made its way into the commit.

Reply:
💯 my bad.